packages/@o3r/testing/schematics/ng-add/fixture/index.ts

```diff
@@ -55,7 +55,7 @@ export function updateFixtureConfig(options: { projectName?: string | null | und
       }
       tsconfigCompilerOptions.esModuleInterop = true;
       tsconfigCompilerOptions.outDir = 'test';
-
+      tsconfigFile.include = (tsconfigFile.include || []).push('**/fixture');
```

Comment:
Do we have file named `fixture` only ?

Reply:
it was for a folder actually. Fixtures for services are in a `fixture` folder

Reply:
I would expect `**/*.fixtures?(.*)?.ts` to support:
- .../my-component.fixture.ts
- .../components/fixtures.ts
- .../my-service/fixtures/my-services.fixture.jest.ts
- .../my-service/fixtures/my-services.fixture.jasmine.ts

Reply:
we also need to include the folder because we have barrels inside the fixtures folder